tests/test_conda_subprocess.py

```
@@ -42,6 +43,34 @@ def test_check_output_name(self):
             expected_output,
         )
 
+    def test_check_output_env_path(self):
+        expected_output = "py313"
+        self.assertEqual(
+            check_output("which python", prefix_path=self.env_path, input=None).split("/")[-3],
```

Ruff (0.8.2)

Line too long (95 > 88)

(E501)